content/influxdb/v2.4/get-started/visualize.md

```diff
@@ -11,21 +11,174 @@ menu:
+    There are many different [visualization types](/influxdb/v2.4/visualize-data/visualization-types/)
+    available.
+    For this tutorial, use the default **Graph** visualization.
+4.  Use the query time range selector to select an absolute time range that
+    covers includes the time range of the 
+    [data written in "Get started writing to InfluxDB"](/influxdb/v2.4/get-started/write/#view-the-written-data):
+    **2022-01-01T08:00:00Z** to **2022-01-01T20:00:01Z**.
+
+    1.  The query time range selector defaults to querying data from the last hour
+        (**{{< icon "clock" >}} Past 1h**).
+        Click the time range selector drop-down menu and select **Custom Time Range**.
+
+        {{< expand-wrapper >}}
+        {{% expand "View time range selector" %}}
+{{< img-hd src="/img/influxdb/2-4-get-started-visualize-time-range.png" alt="InfluxDB time range selector" />}}
+        {{% /expand %}}
+        {{< /expand-wrapper >}}
+
+    2.  Use the data picker to select the stop and stop date and time or manually
+        enter the following start and stop times:
+
+        - **Start**: 2022-01-01 08:00:00
+        - **Stop**: 2022-01-01 20:00:01
+    
+    3. Click **{{% caps %}}Apply Time Range{{% /caps %}}**.
+
+5.  Use the **Query Builder** to select the measurement, fields, and tags to query:
+
+    1. In the **{{% caps %}}From{{% /caps %}}** column, select the **get-started** bucket.
+    2. In the **Filter** column, select the **home** measurement.
+    3. In the next **Filter** column, select the **temp** field.
+    4. In the next **Filter** column, select the **room** tag and the **Kitchen** tag value.
+
+6.  Click **{{% caps %}}Submit{{% /caps %}}** to run the query and visualize the
+    results.
+
+    {{< img-hd src="/img/influxdb/2-4-get-started-visualize-query-builder.png" alt="InfluxDB Query Builder" />}}
+
+7. Click **{{< icon "check" >}}** to save the cell and return to the dashboard.
+
+## Create and use dashboard variables
+
+InfluxDB dashboard cells use **dashboard variables** to dynamically change
+specific parts of cell queries.
+The query builder automatically builds queries using the following
+[predefined dashboard variables](/influxdb/v2.4/visualize-data/variables/#predefined-dashboard-variables),
+each controlled by selections in your dashboard:
+
+- `v.timeRangeStart`: Start time of the queried time range specified by the time range selector.
+- `v.timeRangeStop`: Stop time of the queried time range specified by the time range selector.
+- `v.windowPeriod`: Window period used downsample data to one point per pixel in
+  a cell visualization. The value of this variable is determined by the pixel-width of the cell.
+
+### Create a custom dashboard variable
+
+Let's create a custom dashboard variable that we can use to change the field 
+displayed by your dashboard cell.
+
+1.  Select **Settings > Variables** in the left navigation bar.
+
+    {{< nav-icon "settings" >}}
```

Comment:
I have a gear icon and not a wrench icon in my OSS 2.5 nav bar - not sure if this changed from 2.4 to 2.5?? cloud 2 also has a gear icon.